FirebaseStorageUITests/FUIImageViewCategoryTests.m

```diff
@@ -0,0 +1,125 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+@import XCTest;
+
+@import FirebaseStorageUI;
+@import OCMock;
+
+#import "UIImageView+FirebaseStorage.h"
+
+@interface FUIImageViewCategoryTests : XCTestCase
+@property (nonatomic, readwrite) UIImageView *imageView;
+@property (nonatomic, readwrite) SDImageCache *cache;
+@end
+
+@implementation FUIImageViewCategoryTests
+
+- (void)setUp {
+  [super setUp];
+  self.cache = OCMClassMock([SDImageCache class]);
+  self.imageView = [[UIImageView alloc] initWithFrame:CGRectMake(0, 0, 100, 100)];
+}
+
+- (void)tearDown {
+  [super tearDown];
+}
+
+- (void)testItCreatesADownloadTaskIfCacheIsEmpty {
+  FIRStorageReference *ref = OCMClassMock([FIRStorageReference class]);
```

Comment:
I think that they generally prefer `id ref = OCMClassMock...` since technically it's actually not of type `FIRStorageReference`